docs/DESIGN-PATTERNS.md

```diff
@@ -151,7 +151,7 @@ export class ClickMe implements OnDestroy{
 
   }
 
-  onDestroy(){
+  ngOnDestroy(){
```

Comment:
we can go even further here. Pls get rid completely off `$element` and `OnDestroy` and just use `HostListener('click')`